packages/avatar/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@ui-kit/avatar",
+  "version": "0.0.0",
+  "description": "",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
+  "scripts": {
+    "prebuild": "rimraf dist/**",
+    "build": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js -i ./index.js"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@emotion/core": "10.0.22",
```

Comment:
I'm wondering "how easy" it would be here to miss some runtime dep.

Reply:
So far it seems like you get warnings in the build but not failures.

Reply:
I mean within the monorepo it shouldn't matter, as packages get hoisted. I'm talking about consumers of the package. If some runtime dep is not defined, you will probably get an error then.

I guess this would make more sense to do once the source code is moved into its own package.